index.js

```diff
@@ -226,6 +293,78 @@ Massive.prototype.documentTableSql = function(tableName){
   return sql;
 };
 
+
+Massive.prototype.dropDocumentTable = function(table, options, next) {
```

Comment:
Happy to change the name. Though I do disagree that there is no specific functionality as these tables are created specifically as document store tables. Still in this context I completely agree that changing the name to createTable makes sense. 👍